src/validator-integration.js

```diff
@@ -44,7 +44,26 @@ export function loadScript(doc, url) {
   const script = /** @type {!HTMLScriptElement} */ (
     doc.createElement('script')
   );
-  script.src = url;
+  // Make script.src assignment Trusted Types compatible for compatible browsers
+  if (self.trustedTypes && self.trustedTypes.createPolicy) {
+    const policy = self.trustedTypes.createPolicy(
+      'validator-integration#loadScript',
+      {
+        createScriptURL: function (url) {
+          // Only allow trusted URLs
```

Comment:
would you mind adding the reasoning why it is OK to be explicit about the domain for validator_wasm. No other AMP Cache hosts validator_wasm so we can assume the explicit cdn domain is `cdn.ampproject.org` instead of using the dynamic `cdn` value from `src/config/urls.js`

Reply:
Just added! Let me know if you would like to edit the wording.